### ISSUE TITLE
Label text does not meet color ratio on selection

### ISSUE THREAD

**Issue by misolori** [Microsoft/vscode#56484]
Reverted via https://github.com/Microsoft/vscode/commit/b887217768cc2bca5892dbb054fd5a5aba1dbcf3 in #56465

This appears in the explorer and panels. We should look into changing the opacity on selection.

Explorer
![image](https://user-images.githubusercontent.com/35271042/44160960-d9767300-a070-11e8-8453-4cfeaeea78ac.png)

Panels
![image](https://user-images.githubusercontent.com/35271042/44160971-e004ea80-a070-11e8-9141-30f01f589078.png)

![image](https://user-images.githubusercontent.com/35271042/44161048-0cb90200-a071-11e8-8e6e-0e109ef00245.png)

**Comment by isidorn** [comment]
Changing the opacity on selection makes perfect sense to me.
In the tree we already do something similar we change the text color on selection, so changing the opacity of the description would not be so harsh imho.

fyi @bpasero

**Comment by bpasero** [comment]
Yeah works for me, only on selection is fine.

**Comment by isidorn** [comment]
@misolori selected tree items have the css class `.selected` so this should be easily doable.

**Comment by misolori** [comment]
Bumped up the opacity on selection and this now passes.

![image](https://user-images.githubusercontent.com/35271042/45230477-d7718f80-b27d-11e8-96a8-23301ed0135a.png)

<img width="40%" src="https://user-images.githubusercontent.com/35271042/45230535-fec85c80-b27d-11e8-9bc9-190fe9beb3bb.png">

![image](https://user-images.githubusercontent.com/35271042/45230585-23bccf80-b27e-11e8-8c43-03c9d61c3b8c.png)

<img width="40%" src="https://user-images.githubusercontent.com/35271042/45230611-3a632680-b27e-11e8-9fea-9869349c3504.png">
